### ISSUE TITLE
innosend: message to more than one person

### ISSUE THREAD

**Issue by GoogleCodeExporter** [satyajit319/websmsdroid#48]
```
at the moment it isnt possible to send one sms to more than on person. for
that it is 
important to set the "mass" and "date/time" option in your innosend connector.

```

Original issue reported on code.google.com by `felix.bechstein` on 5 Jan 2010 at 2:47

**Comment by GoogleCodeExporter** [comment]
```
please try if this is fixed with the attached apk.
i don't have the innosend balance to check that :(

btw: don't forget that mass-sms is not available with all types of innosend 
connectors.
```

Original comment by `felix.bechstein` on 6 Jan 2010 at 4:33
- Changed state: **Started**

Attachments:
- [AndGMXsms-release.apk](https://storage.googleapis.com/google-code-attachments/websmsdroid/issue-48/comment-1/AndGMXsms-release.apk)

**Comment by GoogleCodeExporter** [comment]
```
the above fix is obsolete now.

built an other fix working with all innosend types with the fix of issue #50
```

Original comment by `felix.bechstein` on 7 Jan 2010 at 7:53
- Changed state: **Fixed**
